recipes-kernel/kmod/kmod_%.bbappend

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: Andrei Gherzan <andrei@gherzan.com>
+#
+# SPDX-License-Identifier: MIT
+
+PACKAGECONFIG:append:rpi = " xz"
```

Comment:
this will enable it for target but the real problem is `kmod-native` so we need 
```
PACKAGECONFIG:append = " xz"
```

but this will change the generic options and the layer will lose yocto compliance. So best is to enable it in main recipe. So please lobby for the [patch](https://lists.openembedded.org/g/openembedded-core/message/164932) I have sent :)

Reply:
even enabling `xz` for  kmod-native is not resolving this :( so something more is amiss

Reply:
I was just about to do this. Thanks for handling it.